plugins/node/opentelemetry-instrumentation-mongodb/test/mongodb.test.ts

```diff
@@ -249,6 +250,66 @@ describe('MongoDBInstrumentation', () => {
     });
   });
 
+  describe('when specifying a dbStatementSerializer configuration', () => {
```

Comment:
I think we should add a test for the `enhancedDatabaseReporting: false` case as well.

Reply:
@blumamir `enhancedDatabaseReporting` isn't doing anything when the `dbStatementSerializer` is provided, so it isn't necessary to test it in this case

Reply:
I've simply remove it from the test